library/analytics/src/main/java/org/cru/godtools/analytics/appsflyer/AppsFlyerModule.kt

```diff
@@ -0,0 +1,21 @@
+package org.cru.godtools.analytics.appsflyer
+
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import dagger.multibindings.IntoSet
+import dagger.multibindings.Multibinds
+import org.ccci.gto.android.common.dagger.eager.EagerSingleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+abstract class AppsFlyerModule {
+    @Binds
+    @IntoSet
+    @EagerSingleton(threadMode = EagerSingleton.ThreadMode.MAIN)
+    abstract fun AppsFlyerAnalyticsService.eagerSingleton(): Any
+
+    @Multibinds
+    abstract fun deepLinkResolvers(): Set<AppsFlyerDeepLinkResolver>
```

Comment:
Is this binding your deepLinResolvers to AppsFlyer so we can use it?

Reply:
yes, it's our DI (Dependency Injection) framework.

https://dagger.dev/dev-guide/multibindings.html

Reply:
DI & Dagger are a pretty deep topic, for now you can skip over Dagger stuff in you want.

Reply:
we talked about Dagger & DI, so I'm going to resolve this